src/main/java/org/jenkinsci/test/acceptance/junit/JenkinsAcceptanceTestRule.java

```diff
@@ -38,6 +40,18 @@
  * @author Kohsuke Kawaguchi
  */
 public class JenkinsAcceptanceTestRule implements MethodRule { // TODO should use TestRule instead
+
+    private boolean enableInstallWizard;
```

Comment:
That would would force every test to check if they are running on a LocalController or not and  decide if the test should be skipped. By doing it this way we centralice it and make the decision transparent to the test.